client/client_test.go

```diff
@@ -340,7 +340,7 @@ func TestNegotiateAPIVersion(t *testing.T) {
 	}
 }
 
-// TestNegotiateAPIVersionOverride asserts that we honor the DOCKER_API_VERSION
+// TestNegotiateAPVersionOverride asserts that we honor the DOCKER_API_VERSION
 // environment variable when negotiating versions.
 func TestNegotiateAPVersionOverride(t *testing.T) {
```

Comment:
I think we should change `TestNegotiateAPVersionOverride` to `TestNegotiateAPIVersionOverride `  to make the function name more readable here.  Same on line https://github.com/moby/moby/pull/47731/files#diff-57cdc3e29d0ef725a50897a4fbd1c4f76353429fa66d3a785dcbc17dcd7b6f31R359